### ISSUE TITLE
kill current processes on sorry

### ISSUE THREAD

**Issue by cg505** [ocf/utils#131]
When an account is sorried, any current processes running as that user should be `kill -9`d on any machines they could potentially be running on (`tsunami`, `death`, `vampires`, `segfault`, `corruption`, and maybe more?).

Also, user crontabs should be disabled on these machines. Cron won't be able to run anything as the user but will still try, filling our logs with garbage. A recent compromise included adding stuff to crontab.

**Comment by axmmisaka** [comment]
In the new sorry we can just shell out and `kill -SIGKILL` everything with `root`/kerberos ticket, any suggestions on how to restore their cron? Just save them in sorried user's home dir...?

**Comment by cg505** [comment]
yeah, we can do something like with .oldshell, and use crontab -l > ~/.oldcron
